opm/autodiff/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -20,7 +20,7 @@
 #ifndef OPM_WELLSTATEFULLYIMPLICITBLACKOIL_HEADER_INCLUDED
 #define OPM_WELLSTATEFULLYIMPLICITBLACKOIL_HEADER_INCLUDED
 
-
+#include <opm/autodiff/BlackoilModelEnums.hpp>
```

Comment:
I don't understand the changes to this file:
 - Why add the include?
 - What is the new resize() for?